src/BlazorSignalR/Internal/BlazorHttpConnectionFactory.cs

```diff
@@ -42,9 +42,9 @@ public BlazorHttpConnectionFactory(IOptions<BlazorHttpConnectionOptions> options
             }
         }
 
-        public Task DisposeAsync(ConnectionContext connection)
+        public async Task DisposeAsync(ConnectionContext connection)
```

Comment:
There is no need for this change

Reply:
Dispose async returns ValuesTask, this was the easiest conversion i knew. Is there another way?